maven-java/kalix-maven-plugin/src/main/java/kalix/DeployMojo.java

```diff
@@ -36,32 +35,51 @@ public class DeployMojo extends AbstractMojo {
     @Parameter(property = "kalixContext")
     private String kalixContext;
 
+    @Parameter(property = "kalixProject")
+    private String kalixProject;
+
     @Parameter(defaultValue = "30000", property = "cliTimeoutMs", required = true)
     private Long cliTimeoutMs;
 
     private final Log log = getLog();
 
     /**
-     * We deploy by invoking the services deploy command
+     * We deploy by invoking the services deploy command to 'kalixProject' if set.
+     * If 'kalixProject' is not set then deploying to the currently selected project.
      */
     public void execute() throws MojoExecutionException {
-        log.info("Deploying project to Kalix");
+        final List<String> commandLine;
+        final int deploymentResult = 0;
+        if (kalixProject.isEmpty()) {
+            log.info("`kalixProject` hasn't been set. Therefore, deploying to the currently selected project configured via Kalix CLI");
+            return;
```

Comment:
Didn't we say that this should be ok and use the default (so no `--project` parameter to the cli)

Reply:
I was missing the deploy command. Added now